src/components/Activities/Activity3/Game/FeaturesTutorial.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext, useState } from 'react';
+import { GameContext } from '.';
+import { VARIABLE_CONTENT, VARIABLES } from './GameConstants';
+import InfoCard from './InfoCard';
+
+function FeaturesTutorial(): JSX.Element {
+  const { goNextState } = useContext(GameContext);
+  const [index, setIndex] = useState(0);
+  const variableIndexToSlideIndex = [0, 2, 4, 1, 3, 5];
+  const descriptionNames = [VARIABLES.CREDIBLE, VARIABLES.POPULAR, VARIABLES.RECENT_UPLOAD,
+    VARIABLES.SAME_CREATOR, VARIABLES.SAME_CONTENT, VARIABLES.SUBSCRIBED];
+
+  const moveNext = () => {
+    if (index < descriptionNames.length - 1) {
+      setIndex(index + 1);
+    } else {
+      goNextState();
+    }
+  };
```

Comment:
let's change this to useEffect w index because it's easier to see what's going on/less nested
```suggestion
    useEffect(() => {
    if (index >= descriptionNames.length){
      goNextState();
    }
  }, [index]);
```

Reply:
this gives me out of bounds error now. Like it tries to render out of bounds array before the useEffect kicks in